glossary.md

```diff
@@ -206,6 +208,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | orthogonal         | trực giao          |                       |
 | orthonormal        | trực chuẩn         |                       |
 | overfit            | quá khớp           | http://bit.ly/2BvfPYA |
+| one-hot encoding   | mã hoá one-hot     |                       |
```

Comment:
Chỗ này cần team cân nhắc lại. Khi dịch ra tiếng việt thì encode và encrypt đều dịch ra là mã hóa. Nhưng đây là 2 hành động có động cơ và kỹ thuật rất khác nhau.

Encode là chuyển dữ liệu từ định dạng này qua định dạng khác mà vẫn bảo toàn thông tin.
Encrypt là mã hóa dữ liệu để người không được phép không đọc được (không có chìa khóa).

Chữ encode có thể dịch là `biểu diễn` hoặc `số hóa`.

Reply:
Và thật ra, trong project này không nên có chữ "mã hóa" nào cả.

Reply:
+1 cho `biểu diễn`

Reply:
```suggestion
| one-hot encoding   | biễu diễn one-hot     |                       |
```